README.md

```diff
@@ -13,15 +13,14 @@ Included:
 
 
 ## To unbox:
-- Go to the desired location where you would normally run `truffle init`
 - Run `truffle unbox airswap/fluidity-truffle-box`
 
 ## Circle CI:
 A circle CI configuration file is provided already in the repository. This file is set up to perform:
 - `yarn install`
 - test migrations
 - run all tests and calculate their coverage
-- linting (this does not fix linting issues but reports them) 
+- linting (this does not fix linting issues but reports them)
```

Comment:
Add another section to the readme about eth gas reporter - what it does, how you run it etc

Reply:
It presents gas used in tests, however, you don't need to run anything. The configs are in the truffle.js file such that it runs when you run truffle test

Reply:
never mind, i see what you meant. adding a line now.